### PR TITLE
[inih] Fix  Issue 68332 

### PR DESCRIPTION
**Pull request by pkillarjun**
Fix issue [68332](https://bugs.chromium.org/p/oss-fuzz/issues/detail?id=68332) .

### CODE REVIEW

File: projects/inih/Dockerfile
Comment (pkillarjun) [inline projects/inih/Dockerfile:18]: I will update this after https://github.com/benhoyt/inih/pull/168 is merged.

**Comment by pkillarjun** [comment]
summon @DonggeLiu

**Comment by DonggeLiu** [comment]
Double-checking if I understand this correctly:
The `inch` maintainer is still interested in using OSS-Fuzz and receiving crash reports but does not want to maintain the fuzz target and build script in the `inch` repo.
You moved them to your own repo instead.

Also, could you please add a link to `issue 68332` mentioned in the body of this PR?

**Comment by pkillarjun** [comment]
>The `inch` maintainer is still interested in using OSS-Fuzz and receiving crash reports but does not want to maintain the fuzz target and build script in the `inch` repo.
You moved them to your own repo instead.

It's `inih`, 

Yes,  Thanks to my moronic behaviour and Jia Tan(pseudonym).

**Comment by DonggeLiu** [comment]
I see, thanks for confirming this.
Would you mind moving the files from your fork to this repo, e.g., like this?
https://github.com/google/oss-fuzz/tree/master/projects/abseil-cpp

Thanks!

**Comment by pkillarjun** [comment]
> Would you mind moving the files from your fork to this repo, e.g., like this?
https://github.com/google/oss-fuzz/tree/master/projects/abseil-cpp

I can, but I don't want to. This will make the oss-fuzz repo bloated.

**Comment by pkillarjun** [comment]
I have talked to the GitHub team; they said everything is fine with [`0x34d` ](https://github.com/0x34d).

**Comment by DonggeLiu** [comment]
> I can, but I don't want to. This will make the oss-fuzz repo bloated. 

Yep, it will make `OSS-Fuzz` bloated. Hence, our first preference is to host those files in the project-under-test.
If that is not possible, hosting them in `OSS-Fuzz` (like many other projects) is our second preference for safety concerns.
We do appreciate your effort in helping integrate many projects.

> Also, I will think about moving the `build.sh` back into the repo after proper confirmation from Ben Hoyt.

If possible, that would be our first preference. Meanwhile, we can always land the second preference now to unblock the project and improve later.

**Comment by pkillarjun** [comment]
@DonggeLiu  you can merge this PR now;

Dude, 'inih' is only one file; I can bet there won't be any crashes anytime soon.
And I'm exhausted on this one, for fail-safe I like this [philosophy](https://www.youtube.com/watch?v=LgmL4TQCNtI)."

**Review by DonggeLiu — APPROVED** [review]

**Pull request merged**
